### DISCUSSION TITLE
NAB Transact Payment Gateway not displaying

### DISCUSSION THREAD

**Discussion by benryandev** [woocommerce/woocommerce#45037] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?

Beautiful layout compared to the standard WooCommerce checkout experience.

## What do you think is missing from the Cart & Checkout blocks?

Customers site uses NAB Transact which is purchased directly from WooCommerce (https://woocommerce.com/products/nab-transact-direct-post/) as their payment gateway, but WooCommerce Blocks Checkout page just displays PayPal. Can we please add this to the supported gateways considering it's a item purchasable directly through WooCommerce?

**Comment by coolqoo** [comment]
Looks like all 3rd party payment plugins are not displaying

**Comment by ghost** [comment]
Indeed. Same with the WooCommerce Mollie plugin.

**Comment by benryandev** [comment]
Is this likely to be resolved within the next update? I noticed in version 4.4.2 changelog Bug Fixes "Fixes conflicts with 3rd Party payment method integrations."

**Comment by nerrad** [comment]
Hi there folks 👋 

Thanks for taking the time to make us aware that various payment methods aren't working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it's _expected_ that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks. 

Currently it's up to payment method extension developers to add support for the new checkout flow offered via the cart and checkout blocks. We have some [documentation on that here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/extensibility/payment-method-integration.md) and will be publicising this more in the coming months. 

Your best course of action right now if you are using a payment method that isn't working with the blocks is to notify the developer of the extension and ask if they can add support.

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/).
